tools/test/examples/examples.json

```diff
@@ -4,465 +4,1261 @@
       "name": "mbed-os-example-blinky",
       "github": "https://github.com/ARMmbed/mbed-os-example-blinky",
       "mbed": [
-          "https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-blinky"
+        "https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-blinky"
       ],
       "test-repo-source": "github",
-      "features" : [],
-      "targets" : [],
-      "toolchains" : [],
-      "exporters": [],
-      "compile" : true,
-      "export": true,
-      "test" : true,
+      "features": [],
       "baud_rate": 9600,
-      "compare_log": ["mbed-os-example-blinky/tests/blinky.log"],
-      "auto-update" : true
+      "compare_log": [
+        "mbed-os-example-blinky/tests/blinky.log"
+      ],
+      "auto-update": true,
+      "pull_request_test": {
+        "targets": ["NUCLEO_L073RZ"],
+        "toolchains": [],
```

Comment:
Am I reading this properly - if a target is defined, what t oolchains are being t ested if array is empty, all ? same for exporters?

Not in Readme above

Reply:
[] means all possible alternatives